src/main.rs

```diff
@@ -15,56 +15,90 @@ fn modload_handler(dbg: &mut Debugger, modname: &str, base: usize) {
 }
 
 fn main() {
+
+    // mesos.exe -p pid mesos_file0 mesos_file1 mesos_file2 
+    // or mesos.exe mesos_file0 mesos_file1 mesos_file2 -- ./exe arg0 arg1 arg2
+
     // Usage and argument parsing
     let args: Vec<String> = std::env::args().collect();
     if args.len() < 2 {
-        print!("Usage: mesos.exe <pid> [--freq | --verbose | --print] \
-               <explicit meso file 1> <explicit meso file ...>\n");
+        print!("Usage: mesos.exe -p <pid> <options> <mesos files>\n");
+        print!("    or mesos.exe <options> <mesos files> -- program.exe <arguments>\n");
         print!("    --freq               - \
                Treats all breakpoints as frequency breakpoints\n");
         print!("    --verbose            - \
                Enables verbose prints for debugging\n");
         print!("    --print              - \
                Prints breakpoint info on every single breakpoint\n");
+        print!("    --follow-fork        - \
+               Capture coverage for child processes\n");
         print!("    [explicit meso file] - \
                Load a specific meso file regardless of loaded modules\n\n");
 
-        print!("Standard usage: mesos.exe <pid>\n");
         return;
     }
 
-    // Attach to process
-    let mut dbg = Debugger::attach(args[1].parse().unwrap());
 
-    // Register callback routine for module loads so we can attempt to apply
-    // breakpoints to it from the meso file cache
-    dbg.register_modload_callback(Box::new(modload_handler));
+    let mut pid: i32 = -1;
```

Comment:
This should be an `Option<i32>` rather than overloading `-1` to have a "special" meaning